modules/gitops-release-notes-1-2.adoc

```diff
@@ -0,0 +1,132 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
```

Comment:
Please note, we need to add an include statement in the Assembly here: https://raw.githubusercontent.com/openshift/openshift-docs/main/cicd/gitops/gitops-release-notes.adoc
On top of this line: include::modules/gitops-release-notes-1-1.adoc[leveloffset=+1],  in the above file, add: include::modules/gitops-release-notes-1-2.adoc[leveloffset=+1]
Also add a line break between the two include statements.